libs/server/Resp/AdminCommands.cs

```diff
@@ -488,6 +564,104 @@ private bool ProcessAdminCommands<TGarnetApi>(RespCommand command, ReadOnlySpan<
             return true;
         }
 
+        bool EqualsIgnoreCase(ReadOnlySpan<byte> left, ReadOnlySpan<byte> right)
```

Comment:
>EqualsIgnoreCase [](http://example.com/codeflow?start=13&length=16)

This could move to a utils class under libs\common.

Reply:
done, made extension method for even better clarity. pls check.